CHANGELOG.md

```diff
@@ -62,7 +70,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Provider
 - Subscriber
 - Encounter
-- Eligibility 
+- Eligibility
 - Authentication
 - Configuration
 
```

Comment:
Sorry, I forgot to fix these in my release, do you mind adding `0.9.0` and yours too?

Reply:
@mmcrockett Done!